semgrep-rules/adderror-must-be-followed-by-return.go

```diff
@@ -0,0 +1,92 @@
+package provider
+
+// TODO: This file isn't actually catching the ruleid cases,
+// even though the rule works in the actual code correctly.
```

Comment:
Does this mean the semgrep tests don't pass?

Reply:
Yeah, I'm trying to figure out why I can't annotate the tests correctly

Reply:
This may be old news but I always find https://semgrep.dev/playground/new helpful

Reply:
I've found when it's evaluating multi lines putting the annotation above the test function to work https://github.com/stytchauth/api/blob/main/semgrep-rules/unclosed-transaction.go#L13